### ISSUE TITLE
Focus on a already open alertify?

### ISSUE THREAD

**Issue by stoivo** [MohammadYounes/AlertifyJS#79]
Have stored the reference to all open allertifyes in a object. And when the user tries to open the the same that he already have open. Than I want to focus the already open one. If is use `.show()` it resized and moves to its origin position?

What can I do to fix this?

**Comment by MohammadYounes** [comment]
Do you mean something like bring to front ?

**Comment by stoivo** [comment]
Yes :sparkles: sound like have a solution (and maybe got this question before)

**Comment by MohammadYounes** [comment]
The functionality is [there](https://github.com/MohammadYounes/AlertifyJS/blob/master/build/alertify.js#L730-L758), but it's not exposed.

**Comment by MohammadYounes** [comment]
Till this get released, you can simply focus the dialog element, this will trigger the bring to front functionality.

```js
if(dialogInstance.isOpen())
    dialogInstance.elements.dialog.focus();
else
    dialogInstance.show();
```

Thanks!

**Comment by stoivo** [comment]
Can I use https://github.com/MohammadYounes/AlertifyJS/blob/master/build/alertify.min.js should it work than?

**Comment by MohammadYounes** [comment]
It will work with the version you are already using, since it's already there from the start!

**Comment by stoivo** [comment]
ok... Than I am doing something wrong. 
My code look something like 
```js
alertify.dialog('Comment', function factory(){
  return{
    main:function(bank_transaction_id){
      this.message = I18n.t("other_documents.loading");
      window.comment_dialogs[bank_transaction_id] = this;
      // ...
    },

    setup:function(){
        return {
          buttons:[
            { text: I18n.t("cancel"), key:27/*Esc*/},
          ],
          options: {  title: I18n.t("other_documents.loading")},
          focus: { element:1 }
        };
    },

    prepare:function(){
      this.setContent(this.message);
    }
  };
}, true);
```

![screen shot 2015-11-03 at 17 49 38](https://cloud.githubusercontent.com/assets/3492040/10914854/831ebbd0-8254-11e5-905c-8a0004c0ebbd.png)
![screen shot 2015-11-03 at 17 50 07](https://cloud.githubusercontent.com/assets/3492040/10914857/860e892e-8254-11e5-9d3a-e1ea5f59c8f2.png)
![screen shot 2015-11-03 at 17 51 35](https://cloud.githubusercontent.com/assets/3492040/10914861/884f72ac-8254-11e5-8781-cc2cf3e7e597.png)

I can't get it to work

**Comment by MohammadYounes** [comment]
It's working but you can't see it because the actual focus takes place once the main window becomes active. try undocking the dev. tools and return to main window after you execute the focus line.

**Comment by stoivo** [comment]
ok, I created a codepen http://codepen.io/stoivo/pen/dYjaWZ?editors=101

What do I do wrong with bringToFront?

**Comment by MohammadYounes** [comment]
The 1st link `window.a.bringToFront()` will not work, since the method is not exposed yet! The 2nd link `window.a.elements.dialog.focus();` is working fine, I placed *You!* dialog on top of *ME!* dialog then clicked on *show me on top2*  (tested on chrome, FF and IE).

**Comment by stoivo** [comment]
yes, a copy past error

**Comment by stoivo** [comment]
thx, Now I made it work

**Comment by MohammadYounes** [comment]
New option [`bringToFront`](http://alertifyjs.com/alert/bringToFront.html) available as of 1.6 

Thanks!
